WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -535,13 +569,19 @@ private extension CollectOrderPaymentUseCase {
 
     /// Allow merchants to print or email locally-generated receipts.
     ///
-    func presentLocalReceiptAlert(receiptParameters: CardPresentReceiptParameters,
+    func presentLocalReceiptAlert(receiptParameters: CardPresentReceiptParameters?,
                              alertProvider paymentAlerts: any CardReaderTransactionAlertsProviding<AlertPresenter.AlertDetails>,
                              onCompleted: @escaping () -> ()) {
         // Present receipt alert
         alertsPresenter.present(viewModel: paymentAlerts.success(printReceipt: { [order, configuration, weak self] in
             guard let self = self else { return }
 
+            guard let receiptParameters = receiptParameters else {
```

Comment:
super nit: (same for L601)
```suggestion
            guard let receiptParameters else {
```

Reply:
👍 fixed in 80ffa32